Libraries/LibWeb/SVG/SVGElement.cpp

```diff
@@ -32,75 +33,92 @@ void SVGElement::initialize(JS::Realm& realm)
 }
 
 struct NamedPropertyID {
-    NamedPropertyID(CSS::PropertyID property_id)
+    NamedPropertyID(CSS::PropertyID property_id, Vector<FlyString> supported_elements = {})
         : id(property_id)
         , name(CSS::string_from_property_id(property_id))
+        , supported_elements(move(supported_elements))
     {
     }
 
     CSS::PropertyID id;
     StringView name;
```

Comment:
Unrelated to your change, just noticed while we're here - this could be a FlyString right? That would make the name comparisons below O(1) in the happy case I think